5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -1,68 +1,88 @@
-# Guidance for users of version 4.0
+# Changes Compared to v4.x
 
-Existing users of the 4.0 version of the standard may find it useful to see some of the key changes in the content of version 5.0 as well some changes in scope and philosophy that took place during the developement of version 5.0.
+## Introduction
 
-## What's new in 5.0
+Users familiar with version 4.x of the standard may find it helpful to review the key changes introduced in version 5.0, including updates in content, scope, and underlying philosophy.
 
-The following sections try and cover the most major changes
+Of the 286 requirements in version 4.0.3, only 11 remain unchanged, while 15 have undergone minor grammatical adjustments without altering their meaning. In total 109 requirements or 38% are no longer separate requirements in verion 5.0 with 50 simply being deleted, 28 removed as duplicates and 31 merged into other requirements. The rest have been revised in some way. Even requirements that were not substantively modified will have new numbering due to reordering or restructuring.
```

Comment:
" In total, 109 requirements (38%)"?

Reply:
"will have new numbering" > they already have

Reply:
modified